packages/specs/schema/src/components/default/nextMapper.ts

```
@@ -0,0 +1,87 @@
+import {type Type} from "@tsed/core";
+
+import {JsonSchema} from "../../domain/JsonSchema.js";
+import type {JsonSchemaOptions} from "../../interfaces/JsonSchemaOptions.js";
+import {execMapper, registerJsonSchemaMapper} from "../../registries/JsonSchemaMapperContainer.js";
+import {getGenericsOptions} from "../../utils/generics.js";
+import {mergeSchema} from "../../utils/mergeSchema.js";
+
+function getMapper(input: JsonSchema, mapper: string) {
+  return input && input.isClass ? mapper || "class" : "any";
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)